msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -260,6 +261,21 @@ interface IMultipleAccountApplicationCreatedListener {
         void onError(final MsalException exception);
     }
 
+    /**
+     * Listener callback for asynchronous initialization of INativeAuthPublicClientApplication object.
+     */
+    interface INativeAuthApplicationCreatedListener {
```

Comment:
Wondering if this could be moved to `NativeAuthPublicClientApplication` or `INativeAuthPublicClientApplication`

Reply:
It really should